### PR TITLE
RHSTOR-2124: Add support for documentation links in status card.

### PR DESCRIPTION
**Pull request by rexagod**
Add support for documentation links in status card, in the dashboard, in addition to minor design revamps.

Signed-off-by: Pranshu Srivastava <rexagod@gmail.com>

***

Refer: https://issues.redhat.com/browse/RHSTOR-2124

### CODE REVIEW

**Comment by rexagod** [comment]
[Screencast from 08-22-2022 11:42:48 PM.webm](https://user-images.githubusercontent.com/33557095/185990316-521349d9-abb8-4ab3-bfe1-bb61d29535b6.webm)

**Comment by rexagod** [comment]
#### When there's no documentation link present.

[Screencast from 08-22-2022 11:49:16 PM.webm](https://user-images.githubusercontent.com/33557095/185991382-5c5634df-09e4-4483-9f4e-45b99bd1b00b.webm)

**Comment by jhadvig** [comment]
@rexagod thanks for the PR. Could you please link the jira story to this PR.
Also would be good to fix the failing CI.

File: frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx [hunk 1]
```diff
@@ -28,21 +30,34 @@ const getSeverityIcon = (severity: string) => {
...
           <div
             className="co-health-card__alert-item-timestamp co-status-card__health-item-text text-secondary"
             data-test="timestamp"
           >
             {timestamp && <Timestamp simple timestamp={timestamp} />}
           </div>
           <span className="co-status-card__health-item-text co-break-word">{message}</span>
+          {documentationLink && (
+            <a className="co-status-card__alert-item-doc-link" href={documentationLink}>
+              {t('console-shared~Go to documentation')}
```
Comment (TheRealJon) [inline frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx:58]: I think `t` is undefined here and is causing build failures in CI:

```
ERROR in /go/src/github.com/openshift/console/frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx
ERROR in /go/src/github.com/openshift/console/frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx(58,16):
TS2304: Cannot find name 't'.
```

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by nehaberry** [comment]
label /qe-approved

**Comment by rexagod** [comment]
Since there's a requirement to disallow documentation labels in the `PrometheusRule` CR when in a managed services environment, we are using metric relabeling to filter them out whenever `ocs-operator` is deployed from the addon, for which this needs to be a label, rather than an annotation.

**Comment by rexagod** [comment]
Tested that the links are being generated dynamically in the expected format, will ask doc team to confirm the visual below and approve the PR.

[Screencast from 09-07-2022 02:02:46 AM.webm](https://user-images.githubusercontent.com/33557095/188733014-4dd1c315-336c-4e93-ad7c-f363207a4cb6.webm)

File: frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx [hunk 2]
```diff
@@ -28,21 +30,35 @@ const getSeverityIcon = (severity: string) => {
...
+          {name && <span className="co-status-card__alert-item-header">{name}</span>}
           <div
             className="co-health-card__alert-item-timestamp co-status-card__health-item-text text-secondary"
             data-test="timestamp"
           >
             {timestamp && <Timestamp simple timestamp={timestamp} />}
           </div>
           <span className="co-status-card__health-item-text co-break-word">{message}</span>
+          {documentationLink && (
+            <a className="co-status-card__alert-item-doc-link" href={documentationLink}>
```
[the remark this answers is not in the document]
Comment (rexagod) [reply frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx:58]: Done: https://github.com/openshift/console/pull/12059#issuecomment-1274305385.

**Pull request merged**